client/src/templates/Challenges/utils/frame.ts

```diff
@@ -378,8 +378,15 @@ function writeToFrame(content: string, frame?: FrameDocument) {
 }
 
 const writeContentToFrame = (frameContext: Context) => {
+  let doctype = '';
+
+  if (frameContext.sources.contents) {
+    doctype =
+      frameContext.sources.contents.match(/^<!DOCTYPE html>/i)?.[0] || '';
+  }
```

Comment:
```suggestion
  const doctype =
    frameContext.sources.contents?.match(/^<!DOCTYPE html>/i)?.[0] || '';
```
I think the one-liner is clear enough, here. It's a style nit, though, so it doesn't need to block the PR.

Reply:
The one-liner is cool with me.